### PR TITLE
Minimap/orbit icons resprite and remake

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Visually redoes how minimap and orbit icons look.

The ones that were redone look like this (old ones on the right, new ones on the left)

<details>

image

</details>

Adds support and new icons for all factions that have special huds, adds yautja unique icons, all icons for jobs that lack it, so on.

All sprites by me.

# Explain why it's good for the game

Current icons are very hard to tell apart, especially orbit icons, some colors are extremely off, orbit icons do not utilize backgrounds at all (it's just plain color fill), also some things there aren't working correctly at all, resulting in many icons being gray mess (shipside)

Here is an example of current icons:
<details>

image

image

hTH7SC46jJ

U3hYykHC7u

</details>

I overhaul them to looks something like this

<details>

chrome_4IcDecN80W

image

image

PYlGchbRLX

</details>

# Testing Photographs and Procedure
<details>

3SCyS7VqhF

vNWh6gJtUV

gXkkL9QxGG

dreammaker_2SkTzjlmAW

dreammaker_zmQ9lWMm3B

</details>

# Changelog
:cl:
add: minimap icons for all factions, roles, etc
code: tweaked how icons are assembled, added support for custom icon background on equipment preset
imageadd: added a lot of new minimap icons
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: tgui/packages/tgui/interfaces/Orbit/index.tsx [hunk 1]
```diff
@@ -520,24 +548,40 @@ const ObservableTooltip = (props: { readonly item: Observable }) => {
 const ObservableIcon = (props: {
   readonly icon: Observable['icon'];
   readonly background_color: Observable['background_color'];
+  readonly background_icon: Observable['background_icon'];
 }) => {
   const { data } = useBackend<OrbitData>();
   const { icons = [] } = data;
-  const { icon, background_color } = props;
+  const { icon, background_color, background_icon } = props;
   if (!icon || !icons[icon]) {
```
Comment (harryob) [inline tgui/packages/tgui/interfaces/Orbit/index.tsx:556]:
```suggestion
  if (!icon || !icons[icon] || !background_icon || !icons[background_icon]) {
```

**Review by harryob — CHANGES_REQUESTED** [review]
for the typescript error

**Review by harryob — APPROVED** [review]

**Pull request merged**